app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,23 @@
+class CommentsController < ApplicationController
+  before_action :set_post
+
+  def create
+    @comment = @post.comments.build(comment_params.merge(user: @current_user))
+
+    if @comment.save
```

Comment:
- [ ] A comment is created on behalf of the currently logged in user (current_user) of the app.
- [ ] Kindly note that this  comments controller should have new method also 
- [ ] Please make sure it has a route also which will be a `GET`.

### Sample:
`GET /posts/:post_id/comments/new(.:format) comments#new`

### Sample 
```ruby

  def new
    @comment = Comment.new
  end
```

Reply:
@Shedrack-Sunday 
Hello Sir, I have already done as you mentioned. Could I submit my project now?

Reply:
I am ready to submit. And I am waiting the response.

Reply:
Dear  Mr. @Shedrack-Sunday,
I waited for about 4 hours to rectify my dashboard, but nothing changed, so I submitted the project.